.github/workflows/push.yml

```diff
@@ -77,6 +80,44 @@ jobs:
           echo "::set-output name=is_semantic_tag::$IS_SEMANTIC_TAG"
           echo "::set-output name=version::$VERSION"
 
+  build-service:
+    runs-on: ubuntu-latest
+    needs:
+      - set-version
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      -
+        name: Build image
+        uses: docker/build-push-action@v2
+        env:
+          VERSION: ${{ needs.set-version.outputs.version }}
+        with:
+          push: false
+          tags: ${{ env.SERVICE_IMAGE_NAME }}:${{ env.VERSION }}
+          file: Dockerfile-service
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: arn:aws:iam::528868401009:role/kbc-templates-api-push-role
+          aws-region: us-east-1
+        env:
```

Comment:
Videl jsem ze to pozralo credentials pro ten S3 upload, to by mohlo fixnout tohle. Jeste je tomu potreba nastavit permissions pro ten OIDC token , na to jsem zapomnel https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services#adding-permissions-settings

https://github.com/keboola/aws-audit-notifications/blob/f3f1fb2e0b4c291eba7f19beac264770f9d1f8fb/.github/workflows/production.yml#L14

Reply:
@ondrejhlavacek tohle jeste je potreba pridat a melo by se to autorizovat a pak az padnout na push do ECR z branch

Reply:
Je to tak, povolil bych pushe ze všech branchí? Minimálně pro vývoj/test. V některech službách to do image registrů posíláme vše.

Reply:
to pak znamená že z PR klidně releasuješ do produkce bez review když na to ta branch má práva. Tohle to umožnuje hezky zablokovat.

Reply:
Jo, dobře, myslel jsem, že by se mohl produkční trigger na deploy služby udělat na nějakej prefix tagu, ale to se taky dá takhle obejít.

Reply:
já právě nevím jestli v ECR permission jde zamknou na nějaký docker tag, to by bylo super. Protože když to nezamkneš tak ti prefix tagu nepomůže když ho budeš moct pushnout z branch